File: Python/Kategori-Bilangan.py
# // Algoritma:
# Menentukan Bilangan Positif, Negatif, atau Nol

def CekBilangan (number):
    number = int(input("Masukan angka apa saja: "))
    if number > 0 :
        print(f"Angka {number} bilangan Positif")
    elif number < 0 :
        print(f"Angka {number} bilangan Negatif")
    else:
        print(f"Angka {number} bilangan Nol")

CekBilangan (10)

File: Bahasa C/Kategori-Bilangan.c
#include <stdio.h>

int main () {
    // Deklarasi Variabel
    int number;

    printf("Masukan bilangan bulat: ");
    // input bilangan bulat
    scanf ("%d", &number);

    // Cek apakah bilangan tersebut Positif, Negatif atau Nol

    if (number > 0 ) {
        printf ("Bilangan %d: positif\n", number);
    }
    else if (number < 0 ) {
        printf ("Bilangan %d: Negatif\n", number);
    }
    else {
        printf ("Bilangan %d : Nol\n", number);

    }
    return 0; 
}